### PR TITLE
test(android_intent_plus): Integration test

### PR DESCRIPTION
**Pull request by Chever-John**
## Description

Now our integration test just supports the Android platform.

## Related Issues

This is a GSoC task, so there are no associated issues for now.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated the version in `pubspec.yaml` and `CHANGELOG.md`.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Review by miquelbeltran — COMMENTED** [review]
I think the new test is fine as it is (just check that it doesn't throw an exception).

Note that there's some dart formatting issues that need to be solved before merge.

Run `flutter format` on the test file

**Comment by miquelbeltran** [comment]
Please rename the test to `set an alarm on weekdays at 9:30pm should not throw`

**Review by miquelbeltran — APPROVED** [review]

File: packages/android_intent_plus/example/integration_test/android_intent_plus_test.dart [hunk 1]
```diff
@@ -50,6 +50,20 @@ void main() {
     }));
   }, skip: !Platform.isAndroid);
 
+  testWidgets('#set an alarm on weekdays at 9:30pm should not throw',
```
Comment (miquelbeltran) [inline packages/android_intent_plus/example/integration_test/android_intent_plus_test.dart:53]: please remove the `#` from the test description

**Review by miquelbeltran — CHANGES_REQUESTED** [review]

File: packages/android_intent_plus/example/integration_test/android_intent_plus_test.dart (hunk 1 above)
Comment (Chever-John) [reply packages/android_intent_plus/example/integration_test/android_intent_plus_test.dart:53]: Sure~

Comment (Chever-John) [reply packages/android_intent_plus/example/integration_test/android_intent_plus_test.dart]: I have removed the # symbol from all tests～

**Comment by miquelbeltran** [comment]
@Chever-John please set the PR to "Ready for review" when ready

**Pull request merged**